docs/reference/config.md

```diff
@@ -145,6 +145,17 @@ commitAuthor:
 commitMessage: "chore(release): this is a custom release message"
 ```
 
+## DirtyFiles
+```{ .yaml .annotate linenums="1" }
+# Allowing specific case sensitive files to allow using uplift to
+# add unedited but changed files as part of your CI
```

Comment:
Defines a list of files that uplift will ignore when checking the status of the current repository for changes. If a change is detected that is not defined in this list, uplift will assume its default behaviour and fail due to the repository being in a dirty state

Reply:
You will also need to update the `docs/static/schema.json` file to include this new field within the `git` config section. I tend to use the description within the config as the description field within the schema